src/options.c

```diff
@@ -347,19 +377,34 @@ PRINCE_USAGE \
 "--rules-stack=SECTION[,..] stacked rules, applied after regular rules or to\n" \
 "                           modes that otherwise don't support rules\n" \
 "--rules-stack=:rule[;..]   same, using \"immediate\" rule(s)\n" \
+"--rules-skip-nop           skip any NOP \":\" rules (you already ran w/o rules)\n" \
 "--incremental[=MODE]       \"incremental\" mode [using section MODE]\n" \
+"--incremental-charcount=N  override CharCount for incremental mode\n" \
 "--mask[=MASK]              mask mode using MASK (or default from john.conf)\n" \
 "--markov[=OPTIONS]         \"Markov\" mode (see doc/MARKOV)\n" \
+"--mkv-stats=FILE           \"markov\" stats file (see doc/MARKOV)\n" \
 "--external=MODE            external mode or word filter\n" \
 JOHN_USAGE_REGEX \
 "--subsets[=CHARSET]        \"subsets\" mode (see doc/SUBSETS)\n" \
+"--subsets-required=N       The N first characters of \"subsets\" charset are\n" \
+"                           the \"required set\" (see doc/SUBSETS)\n" \
+"--subsets-min-diff=N       Minimum unique characters in subset\n" \
+"--subsets-max-diff=[-]N    Maximum unique characters in subset (negative N is\n" \
+"                           relative to word length)\n" \
 "--stdout[=LENGTH]          just output candidate passwords [cut at LENGTH]\n" \
 "--restore[=NAME]           restore an interrupted session [called NAME]\n" \
 "--session=NAME             give a new session the NAME\n" \
 "--status[=NAME]            print status of a session [called NAME]\n" \
 "--make-charset=FILE        make a charset file. It will be overwritten\n" \
+"--reject-printable         reject printable binaries\n" \
 "--show[=left]              show cracked passwords [if =left, then uncracked]\n" \
+"--show=formats             show information about hashes in a file (JSON)\n" \
+"--show=invalid             show lines that are not valid for selected format(s)\n" \
 "--test[=TIME]              run tests and benchmarks for TIME seconds each\n" \
+"--stress-test[=TIME]       loop self tests forever\n" \
+"--no-mask                  used with --test for alternate benchmark w/o mask\n" \
+"--skip-self-tests          skip self tests\n" \
+"--test-full=LEVEL          run more thorough self-tests\n" \
 "                           (if TIME is explicitly 0, test w/o benchmark)\n" \
```

Comment:
Line 408 here, belongs to `--test` and not `--test-full`

Reply:
Fixed.